docker/hashFolder.ts

```diff
@@ -2,7 +2,7 @@ import { createHash } from 'node:crypto'
 import { createReadStream, readdirSync, statSync } from 'node:fs'
 import { join } from 'node:path'
 
-const hashFile = async (file: string): Promise<string> => {
+export const hashFile = async (file: string): Promise<string> => {
```

Comment:
Duplicate of https://github.com/hello-nrfcloud/backend/blob/99ca8ddd4059584fd4b5fcf084beec78276c50c6/cdk/helpers/lambdas/checksumOfFiles.ts#L15C14-L15C31